File: The Core/tennisSet.cs
bool tennisSet(int score1, int score2) 
{
    if ( Math.Min(score1, score2) >= 7 )
    {
        return false;
    }
    
    if ( Math.Min(score1, score2) >= 5 )
    {
        return Math.Max(score1, score2) == 7;
    }
    
    return Math.Max(score1, score2) == 6;
}

File: The Core/At The Crossroads/14. Tennis Set/tennisSet.lua
function tennisSet(score1, score2)
  
    if math.min(score1, score2) >= 7 then
        return false
    end
    
    if math.min(score1, score2) >= 5 then
        return math.max(score1, score2) == 7
    end
    
    return math.max(score1, score2) == 6
end
